soc/soc_legacy/riscv/CMakeLists.txt

```diff
@@ -1,7 +1,5 @@
 # SPDX-License-Identifier: Apache-2.0
 
-add_subdirectory(common)
```

Comment:
note that this will break all non-ported SoCs depending on RISCV_PRIVILEGED, can't new folder be included in the legacy CMakeLists.txt in the mean time?

Reply:
Won't break any? The folder is included for both hwmv1 and hwmv2 here: https://github.com/zephyrproject-rtos/zephyr/pull/68872/files/a98a9af77b53eef1832e6126ada437358da56ae1#diff-2ab6193fa52398b4ebcf872aa830b4d5a7ac7d5713edbbd4499df1af9fcdc57b

Reply:
I see missed it was at top-level soc cmake